io/disk/htx_block_devices.py

```diff
@@ -0,0 +1,175 @@
+#!/usr/bin/env python
+
+# This program is free software; you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation; either version 2 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.
+#
+# See LICENSE for more details.
+# Copyright: 2017 IBM
+# Author: Naresh Bannoth<nbannoth@in.ibm.com>
+# this script run IO stress on block devices for give time.
+
+import os
+import time
+from avocado import Test
+from avocado import main
+from avocado.utils.software_manager import SoftwareManager
+from avocado.utils import build
+from avocado.utils import process, git
+from avocado.utils import distro
+
+
+class HtxTest(Test):
+
+    """
+    HTX [Hardware Test eXecutive] is a test tool suite. The goal of HTX is to
+    stress test the system by exercising all hardware components concurrently
+    in order to uncover any hardware design flaws and hardware hardware or
+    hardware-software interaction issues.
+    :see:https://github.com/open-power/HTX.git
+    :param block_devices: names of block_devices on which you want to run HTX
+    :param mdt_file: mdt file used to trigger HTX
+    :params time_limit: how much time you want to run this stress.
+    """
+
+    def setUp(self):
+        """
+        Build 'HTX'.
+        """
+        self.distro_name = distro.detect().name
+        packages = ['git', 'gcc', 'make', 'libncurses5', 'g++', 'libdapl-dev',
+                    'ncurses-dev', 'libncurses-dev']
+        smm = SoftwareManager()
+        if self.distro_name == 'Ubuntu':
```

Comment:
this seem me used only this place can it like this 

`if distro.detect().name == 'Ubuntu': `  just want to avoid extra variable